src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -303,6 +306,131 @@ void run_WithSyncAsArgumentWithAllArg_ShouldExecuteAllSyncers() {
     }
   }
 
+  @Test
+  void run_WithUpdatedCustomer_ShouldSyncCustomersAndStoreLastSyncTimestampsAsCustomObject() {
+    ZonedDateTime lastModifiedTime = null;
+    // preparation
+    try (final SphereClient targetClient = createClient(CTP_TARGET_CLIENT_CONFIG)) {
+      try (final SphereClient sourceClient = createClient(CTP_SOURCE_CLIENT_CONFIG)) {
+
+        final SyncerFactory syncerFactory =
+            SyncerFactory.of(() -> sourceClient, () -> targetClient, Clock.systemDefaultZone());
+
+        // test
+        CliRunner.of().run(new String[] {"-s", "customers"}, syncerFactory);
+      }
+    }
+
+    // create clients again (for assertions and cleanup), since the run method closes the clients
+    // after execution is done.
+    try (final SphereClient postSourceClient = createClient(CTP_SOURCE_CLIENT_CONFIG)) {
+      try (final SphereClient postTargetClient = createClient(CTP_TARGET_CLIENT_CONFIG)) {
+        // assertions
+        assertThat(syncerTestLogger.getAllLoggingEvents()).hasSize(2);
+
+        assertCustomerSyncerLoggingEvents(syncerTestLogger, 1);
+
+        assertCustomersAreSyncedCorrectly(postTargetClient);
+
+        final String sourceProjectKey = postSourceClient.getConfig().getProjectKey();
+
+        assertLastSyncCustomObjectExists(
+            postTargetClient, sourceProjectKey, "customerSync", "runnerName");
```

Comment:
You don't need these assertions. They're exactly the same as `run_WithCustomerSyncAsArgument_ShouldSyncCustomersAndStoreLastSyncTimestampsAsCustomObject`, so it's already tested in the other test. Additionally, you're not testing those values here. This is just a preparation stage. And also the comment above (`// create clients again (for assertions and cleanup), since...`) does not match what you have here.
So I would remove these assertions.